src/types/Filter.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filter {
```

Comment:
You are already using enums to filter, so you can use the default options to make less data traffic
```js
export enum Filter {
  Completed,
  Active,
  All,
}
```